protocol.md

```diff
@@ -295,6 +295,54 @@ with the report. Others accept if the signature is valid under the included `vk`
 In this scheme the health authorities or anyone else can check the signed
 record, and do not have to be trusted to mitigate impersonation.
 
+## Modified proposal
+
+Roughly, we replace `S` and `L` by the signing and verification keys of a
+signature scheme.
+
+**XXX** Remove the description above after initial review of the new description.
```

Comment:
There should also be a section that sets out terminology:

- **CEN**
- **CEK**: previously "CEN Key"
- **Report**: compressed data that reports exposure related to a set of contact event numbers